components/select_team/select_team.jsx

```diff
@@ -63,20 +63,18 @@ export default class SelectTeam extends React.Component {
         actions.loadRolesIfNeeded(currentUserRoles.split(' '));
     }
 
-    handleTeamClick = (team) => {
+    handleTeamClick = async (team) => {
         this.setState({loadingTeamId: team.id});
 
-        addUserToTeamFromInvite('', team.invite_id,
-            () => {
-                this.props.history.push(`/${team.name}/channels/town-square`);
-            },
-            (error) => {
-                this.setState({
-                    error,
-                    loadingTeamId: '',
-                });
-            }
-        );
+        const {data, error} = await this.props.actions.addUserToTeamFromInvite('', team.invite_id);
```

Comment:
what is the first parameter `token` used for I can't remember...

looking at the api doc it seems inconsistent as it asks for a token but there is nothing about the invite_id https://api.mattermost.com/#tag/teams%2Fpaths%2F~1teams~1%7Bteam_id%7D~1members%2Fpost 

@jwilander thoughts?

Reply:
The documentation is just missing the invite ID, you can use that as well